.github/workflows/backend.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     - name: Build static viz frontend
       run: yarn build-static-viz
     - name: Collect the test coverage
-      run: clojure -X:dev:ee:ee-dev:test:cloverage
+      run: clojure -X:dev:ci:ee:ee-dev:test:cloverage
```

Comment:
Cloverage was timing out on CI because it kept running into OOM errors, but we weren't setting `-Xmx`; this should fix it I think.